pymc_marketing/mmm/components/base.py

```diff
@@ -117,7 +120,9 @@ class Transformation:
     lookup_name: str
 
     def __init__(
-        self, priors: dict[str, Prior] | None = None, prefix: str | None = None
+        self,
+        priors: Prior | float | TensorVariable | None = None,
```

Comment:
Can you please update the type hints in the doctrines 🙏 ?

Reply:
Done!